File: ARCADE/Intro/Level 9_1: growingPlant.cs
    using System;
     
    public class Test
    {
    	public static int growingPlant(int upSpeed, int downSpeed, int desiredHeight){
                return desiredHeight > upSpeed ? desiredHeight / (upSpeed - downSpeed) : 1;
            }
    	public static void Main()
    	{
    		int upSpeed = 100, downSpeed = 10, desiredHeight = 910;
    		Console.WriteLine(growingPlant(upSpeed, downSpeed, desiredHeight));
    	}
    }

File: ARCADE/Intro/Level 9_1: growingPlant.pas
program growingPlant;
 
function growingPlant(upSpeed: integer; downSpeed: integer; desiredHeight: integer): integer;
begin
    if(desiredHeight > upSpeed) then growingPlant := desiredHeight div (upSpeed - downSpeed)
    else growingPlant := 1;
end;
 
var upSpeed, downSpeed, desiredHeight: integer;
begin
	upSpeed := 100; downSpeed := 10; desiredHeight := 910;
	writeln(growingPlant(upSpeed, downSpeed, desiredHeight));
end.
